.gitignore

```diff
@@ -0,0 +1 @@
+*~
```

Comment:
Hehe, I usually rely on gitignore.io to [generate the initial `.gitignore`](https://www.gitignore.io/api/vim,linux,go)

Reply:
Good point!
Didn't know about this service. Added the generated version.